bugtool/cmd/helper.go

```diff
@@ -167,8 +167,8 @@ func hashEncryptionKeys(output string) string {
 	// Search for lines containing encryption keys.
 	// Note that `auth-trunc` is also a relevant pattern, but we already match on
 	// the more generic `auth` pattern.
+	r := regexp.MustCompile("(auth|enc|aead|comp)(.*[[:blank:]](0[xX][[:xdigit:]]+))?")
```

Comment:
nit: Given that `hashEncryptionKeys` itself is called in a loop, would it make sense to extract it globally?